crates/holochain/src/fixt.rs

```diff
@@ -61,16 +61,20 @@ impl Iterator for RealRibosomeFixturator<curve::Zomes> {
             SweetDnaFile::from_test_wasms(uuid, input, Default::default()).await
         });
 
-        let ribosome =
-            RealRibosome::new(dna_file, Arc::new(RwLock::new(ModuleCache::new(None)))).unwrap();
+        let ribosome = tokio_helper::block_forever_on(RealRibosome::new(
```

Comment:
This one I get because it allows parallelizing work inside the constructor but this still blocks the current thread here right?